From 837d59dd57bd742eb19fcc302892ec6c08218c68 Mon Sep 17 00:00:00 2001
From: Xu Mao <B34688@freescale.com>
Date: Thu, 26 Dec 2013 18:51:41 +0800
Subject: [PATCH 277/310] ENGR00133115 device codec into two different
 directory

/device/fsl-codec and
/device/fsl-proprietary/media-profile/

Signed-off-by: Xu Mao <B34688@freescale.com>
---
 arm2_6dq/AndroidBoard.mk       | 3 ++-
 evk_6sl/AndroidBoard.mk        | 3 ++-
 hdmidongle_6dq/AndroidBoard.mk | 3 ++-
 imx53_smd/AndroidBoard.mk      | 3 ++-
 imx5x/BoardConfigCommon.mk     | 2 ++
 imx6/soc/imx6dq.mk             | 2 ++
 sabreauto_6q/AndroidBoard.mk   | 3 ++-
 sabresd_6dq/AndroidBoard.mk    | 3 ++-
 8 files changed, 16 insertions(+), 6 deletions(-)

diff --git a/arm2_6dq/AndroidBoard.mk b/arm2_6dq/AndroidBoard.mk
index 82cf3f5..b70e905 100644
--- a/arm2_6dq/AndroidBoard.mk
+++ b/arm2_6dq/AndroidBoard.mk
@@ -1,6 +1,7 @@
 LOCAL_PATH := $(call my-dir)
 
 ifeq ($(PREBUILT_FSL_IMX_CODEC),true)
-include device/fsl-proprietary/codec/fsl-codec.mk
+-include device/fsl-codec/fsl-codec.mk
 endif
+include device/fsl-proprietary/media-profile/media-profile.mk
 include device/fsl-proprietary/sensor/fsl-sensor.mk
diff --git a/evk_6sl/AndroidBoard.mk b/evk_6sl/AndroidBoard.mk
index 0b14928..885cc95 100644
--- a/evk_6sl/AndroidBoard.mk
+++ b/evk_6sl/AndroidBoard.mk
@@ -1,5 +1,6 @@
 LOCAL_PATH := $(call my-dir)
 ifeq ($(PREBUILT_FSL_IMX_CODEC),true)
--include device/fsl-proprietary/codec/fsl-codec.mk
+-include device/fsl-codec/fsl-codec.mk
 endif
+-include device/fsl-proprietary/media-profile/media-profile.mk
 -include device/fsl-proprietary/sensor/fsl-sensor.mk
diff --git a/hdmidongle_6dq/AndroidBoard.mk b/hdmidongle_6dq/AndroidBoard.mk
index 6d999c1..bc3d1a4 100644
--- a/hdmidongle_6dq/AndroidBoard.mk
+++ b/hdmidongle_6dq/AndroidBoard.mk
@@ -1,5 +1,6 @@
 LOCAL_PATH := $(call my-dir)
 
 ifeq ($(PREBUILT_FSL_IMX_CODEC),true)
-include device/fsl-proprietary/codec/fsl-codec.mk
+-include device/fsl-codec/fsl-codec.mk
 endif
+include device/fsl-proprietary/media-profile/media-profile.mk
diff --git a/imx53_smd/AndroidBoard.mk b/imx53_smd/AndroidBoard.mk
index ee67462..025d1ba 100644
--- a/imx53_smd/AndroidBoard.mk
+++ b/imx53_smd/AndroidBoard.mk
@@ -3,8 +3,9 @@ LOCAL_PATH := $(call my-dir)
 include device/fsl/imx5x/AndroidBoardCommon.mk
 
 ifeq ($(PREBUILT_FSL_IMX_CODEC),true)
-include device/fsl-proprietary/codec/fsl-codec.mk
+-include device/fsl-codec/fsl-codec.mk
 endif
 
+include device/fsl-proprietary/media-profile/media-profile.mk
 include device/fsl-proprietary/gpu/fsl-gpu.mk
 include device/fsl-proprietary/sensor/fsl-sensor.mk
diff --git a/imx5x/BoardConfigCommon.mk b/imx5x/BoardConfigCommon.mk
index 901176d..c1914e6 100755
--- a/imx5x/BoardConfigCommon.mk
+++ b/imx5x/BoardConfigCommon.mk
@@ -41,6 +41,8 @@ TARGET_HAVE_IMX_HWCOMPOSER := true
 USE_ION_ALLOCATOR := true
 
 HAVE_FSL_IMX_CODEC := true
+-include external/fsl_vpu_omx/codec_env.mk
+-include external/fsl_imx_omx/codec_env.mk
 BUILD_WITHOUT_FSL_DIRECTRENDER := false
 BUILD_WITHOUT_FSL_XEC := true
 
diff --git a/imx6/soc/imx6dq.mk b/imx6/soc/imx6dq.mk
index adfb801..98d55a4 100644
--- a/imx6/soc/imx6dq.mk
+++ b/imx6/soc/imx6dq.mk
@@ -10,6 +10,8 @@ HAVE_FSL_IMX_IPU := true
 BOARD_KERNEL_BASE := 0x10800000
 TARGET_KERNEL_DEFCONF := imx6_android_defconfig
 HAVE_FSL_IMX_CODEC := true
+-include external/fsl_vpu_omx/codec_env.mk
+-include external/fsl_imx_omx/codec_env.mk
 TARGET_HAVE_IMX_GRALLOC := true
 TARGET_HAVE_IMX_HWCOMPOSER = true
 USE_OPENGL_RENDERER := true
diff --git a/sabreauto_6q/AndroidBoard.mk b/sabreauto_6q/AndroidBoard.mk
index 4722102..2b28473 100644
--- a/sabreauto_6q/AndroidBoard.mk
+++ b/sabreauto_6q/AndroidBoard.mk
@@ -1,7 +1,8 @@
 LOCAL_PATH := $(call my-dir)
 
 ifeq ($(PREBUILT_FSL_IMX_CODEC),true)
-include device/fsl-proprietary/codec/fsl-codec.mk
+-include device/fsl-codec/fsl-codec.mk
 endif
 
+include device/fsl-proprietary/media-profile/media-profile.mk
 include device/fsl-proprietary/sensor/fsl-sensor.mk
diff --git a/sabresd_6dq/AndroidBoard.mk b/sabresd_6dq/AndroidBoard.mk
index 82cf3f5..b70e905 100644
--- a/sabresd_6dq/AndroidBoard.mk
+++ b/sabresd_6dq/AndroidBoard.mk
@@ -1,6 +1,7 @@
 LOCAL_PATH := $(call my-dir)
 
 ifeq ($(PREBUILT_FSL_IMX_CODEC),true)
-include device/fsl-proprietary/codec/fsl-codec.mk
+-include device/fsl-codec/fsl-codec.mk
 endif
+include device/fsl-proprietary/media-profile/media-profile.mk
 include device/fsl-proprietary/sensor/fsl-sensor.mk
-- 
1.8.0

